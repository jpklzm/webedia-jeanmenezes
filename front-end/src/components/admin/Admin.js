import React from 'react';

import Menu from '../home/menu/Menu';

import styles from './styles.scss';

const Home = () => (
  <div className={styles.container}>
    <h1 className={styles.h1}>Um site do webedia group</h1>
    <Menu />
  </div>
);

export default Home;
