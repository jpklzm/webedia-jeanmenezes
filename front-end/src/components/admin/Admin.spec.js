import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Admin from './Admin';

test('Snapshot', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Admin />
    </MemoryRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
