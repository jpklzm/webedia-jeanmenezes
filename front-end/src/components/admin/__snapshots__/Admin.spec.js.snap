// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div
  className="container"
>
  <div
    id="page-wrap"
  >
    <h1
      className="h1"
    >
      Um site do
      <span
        className="logo"
      >
         
        <strong>
          web
        </strong>
        edia
      </span>
      <span
        className="tm"
      >
        ™
      </span>
       group
    </h1>
    <nav
      className="navbar navbar-default"
      role="navigation"
    >
      <div
        className="container"
      >
        <ul
          className="nav nav-justified"
        >
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="/blog/1"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="#"
            >
              Lorem Ipsum
            </a>
          </li>
          <li
            className="menuItem"
          >
            <a
              className="menuText"
              href="/admin"
              onClick={[Function]}
            >
              Admin
            </a>
          </li>
        </ul>
      </div>
    </nav>
    <div
      className="container"
    >
      <div
        className="container"
      >
        <h1>
           Criar post 
        </h1>
        <form
          onSubmit={[Function]}
        >
          <div>
            <input
              className="input"
              onSubmit={false}
              placeholder="Título"
              type="text"
            />
          </div>
          <div>
            <input
              className="input"
              onSubmit={false}
              placeholder="Subtítulo"
              type="text"
            />
          </div>
          <div>
            <input
              className="input"
              onSubmit={false}
              placeholder="Descrição"
              type="text"
            />
          </div>
          <div>
            <textarea
              className="textarea"
              onSubmit={false}
              placeholder="Conteúdo"
              rows="5"
              type="text"
            />
          </div>
          <div>
            <input
              className="input"
              onSubmit={false}
              placeholder="URL da Imagem"
              type="url"
            />
          </div>
          <button
            className="button"
            type="submit"
          >
            Criar Post
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
